tests/test_runner.py

```diff
@@ -62,5 +63,8 @@ def run_tests(argv, master_config_file=None, global_config_files=None):
     except docker.errors.ImageNotFound as inf:
         print(str(inf))
         return os.EX_CONFIG
+    except python_on_whales.exceptions.DockerException as de:
+        print(str(de))
+        return os.EX_CONFIG
```

Comment:
May need to take this out if I add the failure in its own test to by-pass the timeout issue.